src/runtime/math.py

```diff
@@ -215,25 +227,76 @@ def fabs(x: f64) -> f64:
         return -x
     return x
 
+@overload
+def fabs(x: i64) -> i64:
+    """
+    Return the absolute value of `x`.
+    """
+    if x < 0.0:
+        return -x
+    return x
```

Comment:
This should return a float, no?
```python
>>> import math
>>> math.fabs(4)
4.0
>>> import numpy as np
>>> np.fabs(4)
4.0
```

Reply:
Yes, it was returning me float value only when I ran it locally

Reply:
This is what I get with this PR:
```console
(lp) namannimmo@srf lpython % cat t.py
from math import fabs

def main():
  i: i32
  i = 4
  print(fabs(i))
  print(fabs(4))
  print(fabs(-4))

main()
(lp) namannimmo@srf lpython % python t.py
4.0
4.0
4.0
(lp) namannimmo@srf lpython % lpython t.py && ./a.out 
4
4
4
```

Reply:
Have you tried `float(x)` instead of `x`?